blog/2023-01-10-pulsar-2022-year-in-review.md

```diff
@@ -0,0 +1,109 @@
+- [Flink-Pulsar SQL Connector](https://github.com/streamnative/flink): The Flink-Pulsar SQL Connector supports querying data from and writing data to Pulsar topics with simple Flink SQL queries. Read this [blog](https://streamnative.io/blog/release/2022-09-29-announcing-the-flink-pulsar-sql-connector/) to see details.
+- [Google Cloud Pub/Sub Connector for Apache Pulsar](https://github.com/streamnative/pulsar-io-google-pubsub): It is a Pulsar IO connector enabling data replication between Google Cloud Pub/Sub and Apache Pulsar. Read this [blog](https://streamnative.io/blog/release/2022-6-24-announcing-the-google-cloud-pub-sub-connector-for-apache-pulsar/) to see details.
+- [Google Cloud BigQuery Sink Connector for Apache Pulsar](https://github.com/streamnative/pulsar-io-bigquery): This connector pulls data from Pulsar topics and persists data to Google Cloud BigQuery tables. Read this [blog](https://streamnative.io/blog/release/2022-8-3-announcing-the-google-cloud-bigquery-sink-connector-for-apache-pulsar/) to see details.
+- [Delta Lake Sink Connector for Apache Pulsar](https://github.com/streamnative/pulsar-io-lakehouse): It is a Pulsar IO connector that pulls data from Apache Pulsar topics and persists data to Delta Lake. Read this [blog](https://streamnative.io/blog/release/2022-08-17-announcing-the-delta-lake-sink-connector-for-apache-pulsar/) to see details.
+- [Iceberg Sink Connector for Apache Pulsar](https://github.com/streamnative/pulsar-io-lakehouse): It is a Pulsar IO connector that pulls data from Apache Pulsar topics and persists data to Iceberg tables. Read this [blog](https://streamnative.io/blog/release/2022-12-14-announcing-the-iceberg-sink-connector-for-apache-pulsar/) to see details.
+- [Pulsar Resources Operator for Kubernetes](https://github.com/streamnative/pulsar-resources-operator): The Pulsar Resources Operator is an independent controller that automatically manages Pulsar resources on Kubernetes using manifest files. Read this [blog](https://streamnative.io/blog/release/2022-08-15-introducing-pulsar-resources-operator-for-kubernetes/) to see details.
+
+## What’s next in 2023
+
+The Pulsar community seeks to further improve the project from different aspects. For example, we plan on releasing the first long-term support (LTS) version of Pulsar to meet the needs of different users. For those who want a more stable Pulsar version, they can choose the LTS version, which is expected to be released every 18 months with bug fix support for 24 months and security patches for 36 months. Ordinary releases will come out every 3 months with bug fix support and security patches for 6 months.
```

Comment:
I wonder if it is worth changing the language here so that we do not over promise release frequency. The 2.11 release took 4 months, and it is only just wrapping up now. I know that we've discussed targeting frequent releases, and I know that users like predictability, but I think it is better to under promise release frequency here.

Reply:
Thanks. I removed the frequency info here.